x-pack/plugins/security_solution/server/endpoint/lib/artifacts/migrate_artifacts_to_fleet.test.ts

```diff
@@ -70,6 +111,18 @@ describe('When migrating artifacts to fleet', () => {
     expect(soClient.delete).toHaveBeenCalled();
   });
 
+  it('should create artifact in fleet with attributes that match the SO version', async () => {
+    await migrateArtifactsToFleet(soClient, artifactClient, logger);
+
+    await expect(artifactClient.createArtifact.mock.results[0].value).resolves.toEqual(
+      expect.objectContaining({
+        ...soArtifactEntry,
+        compressionAlgorithm: 'zlib',
+        body: 'eJyrVkrNKynKTC1WsoqOrQUAJxkFKQ==',
```

Comment:
Just noticed this (thanks for pointing it out.
Can you remove this and instead changing the migration change to not mutate the `body` prop? like:

```typescript
if (isCompressed(artifact.attributes)) {
  artifact.attributes = {
    ...artifact.attributes,
    body: (
      await inflateAsync(Buffer.from(artifact.attributes.body, 'base64'))
    ).toString('base64'),
  };
}
```

Or - capture a copy of the `soArtifatEntry` here in the test prior to calling migration (above like 115) and then use that to check again on like 119 (and remove `body` from here)